relay/polkadot/src/lib.rs

```diff
@@ -816,6 +816,12 @@ impl pallet_identity::Config for Runtime {
 	type Slashed = Treasury;
 	type ForceOrigin = EitherOf<EnsureRoot<Self::AccountId>, GeneralAdmin>;
 	type RegistrarOrigin = EitherOf<EnsureRoot<Self::AccountId>, GeneralAdmin>;
+	type OffchainSignature = Signature;
+	type SigningPublicKey = <Signature as Verify>::Signer;
+	type UsernameAuthorityOrigin = EnsureRoot<Self::AccountId>;
```

Comment:
Only with root track? may be plus GeneralAdmin?

Reply:
Optimistically the Kusama People Chain launch goes well and we do Polkadot one release cycle later. But using the feature will just mean more for UIs to migrate. Then yes I think it should be an easier origin for this.

Reply:
So we should use an easier origin here @joepetrowski?

Reply:
I was thinking that we leave it as Root now and then change to something like General Admin (or a new People Admin track) once it is on a parachain, since the feature will be frozen for some time during the migration to a parachain anyway. But if there is demand for this now then I don't have any objections to something easier.